tests/lib/conftest.py

```diff
@@ -1,155 +1,68 @@
-# pylint: disable=missing-function-docstring, redefined-outer-name
-
-import json
-import os
-from datetime import datetime
-from pathlib import Path
-from uuid import uuid4
+"""This file is only for the database fixture. For all other fixtures add them to the
+tests/conftest.py file.
+"""
 
 import pytest
-from fastapi import FastAPI
-from sqlalchemy_utils.functions import create_database, database_exists
-from starlette.testclient import TestClient
-
-from fides.api.ops.api.v1.scope_registry import PRIVACY_REQUEST_READ
-from fides.api.ops.api.v1.scope_registry import SCOPE_REGISTRY as SCOPES
-from fides.core.config import get_config
-from fides.lib.cryptography.schemas.jwt import (
-    JWE_ISSUED_AT,
-    JWE_PAYLOAD_CLIENT_ID,
-    JWE_PAYLOAD_SCOPES,
-)
-from fides.lib.db.base import Base  # type: ignore[attr-defined]
+import requests
+from sqlalchemy.exc import IntegrityError
+
+from fides.api.ops.db.base import Base
 from fides.lib.db.session import get_db_engine, get_db_session
-from fides.lib.models.client import ClientDetail
-from fides.lib.models.fides_user import FidesUser
-from fides.lib.models.fides_user_permissions import FidesUserPermissions
-from fides.lib.oauth.api.routes.user_endpoints import router
-from fides.lib.oauth.jwt import generate_jwe
 from tests.conftest import create_citext_extension
 
-ROOT_PATH = Path().absolute()
 
+@pytest.fixture(scope="session", autouse=True)
+def setup_db(api_client, config):
+    """Apply migrations at beginning and end of testing session"""
+    assert config.test_mode
+    assert requests.post != api_client.post
+    yield api_client.post(url=f"{config.cli.server_url}/v1/admin/db/reset")
 
-@pytest.fixture
-def db(config):
-    """Yield a connection to the test DB."""
-    # Included so that `AccessManualWebhook` can be located when
-    # `ConnectionConfig` is instantiated.
-    from fides.api.ops.models.manual_webhook import (  # pylint: disable=unused-import
```

Comment:
is there any worry to losing this? not sure what it's used for, and if tests are passing it should be fine - but just curious.

Reply:
It seemed to me it was imported by mistake. I never got used for anything.